monsteer/steering/playbackState.fbs

```diff
@@ -3,10 +3,14 @@
 //
 namespace monsteer.steering;
 
+enum State : uint
+{
+  PAUSE
+  PLAY
+}
+
 table PlaybackState
 {
   messageID:string;
```

Comment:
I don't see the messageID being used anywhere and I can't find a future use for keeping it.